dashboards/xdcr-dashboard.json

```diff
@@ -626,6 +626,76 @@
           "_base": "target"
         }
       ]
+    },
+    {
+      "title": "goxdcr mem size and resident",
+      "_base": "panel",
+      "_targets": [
+        {
+          "datasource": "{data-source-name}",
+          "expr": "sysproc_mem_size{proc=\"goxdcr\"}",
+          "legendFormat": "{data-source-name} {{name}} {{proc}}",
+          "_base": "target"
+        },
+        {
+          "datasource": "{data-source-name}",
+          "expr": "sysproc_mem_resident{proc=\"goxdcr\"}",
+          "legendFormat": "{data-source-name} {{name}} {{proc}}",
+          "_base": "target"
+        }
+      ],
+      "fieldConfig": {
+        "defaults": {
+          "custom": {
+            "fillOpacity": 1
+          },
+          "unit": "bytes"
+        }
+      },
+      "options": {
+        "tooltip": {
+          "mode": "multi"
+        }
+      },
+      "type": "timeseries"
+    },
+    {
+      "title": "XDCR Number of running replications",
+      "_base": "panel",
+      "_targets": [
+        {
+          "datasource": "{data-source-name}",
+          "expr": "sum by (status) (xdcr_pipeline_status{status=\"Running\"})",
+          "legendFormat": "{data-source-name} number of running replications}",
+          "_base": "target"
+        }
+      ]
+    },
+    {
+      "title": "goxdcr resident mem size per running replication",
+      "_base": "panel",
+      "_targets": [
+        {
+          "datasource": "{data-source-name}",
+          "expr": "sysproc_mem_resident{proc=\"goxdcr\"} / ignoring(status, category, instance, job, name, proc) (sum by (status) (xdcr_pipeline_status{status=\"Running\"})) ",
+          "legendFormat": "{data-source-name} {{name}} {{proc}}",
+          "_base": "target"
+        }
+      ],
+      "fieldConfig": {
+        "defaults": {
+          "custom": {
+            "fillOpacity": 1
+          },
+          "unit": "bytes"
+        }
+      },
+      "options": {
```

Comment:
Same comment as above.

Reply:
Whoops - erroneously included the unit which is incorrect because you should specify "bytes". ("short" is the default). But "timeseries" and mode of "multi" aren't needed.